component-samples/component-http-client-to0-sample/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+     Copyright 2020 Intel Corporation
+     SPDX-License-Identifier: Apache 2.0
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>component-samples</artifactId>
+    <groupId>org.fido.iot</groupId>
+    <version>0.5-SNAPSHOT</version>
+  </parent>
+
+  <name>Component HTTP TO0 Client Sample</name>
+  <artifactId>component-http-client-to0-sample</artifactId>
```

Comment:
Rename the base folder from component-http-client-to0-sample => to0client
Update artifactId from component-http-client-to0-sample => to0client

It would make it closer to other component sample names.